.github/workflows/continuous-integration.yml

```
@@ -87,6 +87,13 @@ jobs:
         with:
           name: coverage
           path: coverage/
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '5.0.103'
+      - name: Run dotnet tests
+        run: |
+          dotnet test services/Skyra.Tests
```

```suggestion
        run: dotnet test services/Skyra.Tests
```

Please don't do this. You only need a pipe for multi-line scripts.